trade-finance-manager-api/src/v1/api.js

```diff
@@ -820,13 +821,35 @@ const getPartyDbInfo = async ({ companyRegNo }) => {
       url: `${EXTERNAL_API_URL}/party-db/${encodeURIComponent(companyRegNo)}`,
       headers: headers.external,
     });
-    return response.data;
+    return AUTOMATIC_SF_CUSTOMER_CREATION_ENABLED === 'true' ? { status: 200, data: response.data } : response.data;
```

Comment:
So while this means there will be a change that isn't FF'd, is it worth just making this behaviour change regardless of the FF? It's a little surprising to have different return types like this depending on a state of the FF

Reply:
I don't think it should really matter, I'd rather not change the behaviour of the existing endpoint in a feature flagged PR, but I'll add tests to ensure the update in response type depending on the flag if you're happy with that compromise?

Reply:
Mm, yeah I'm sold on that that sounds fine to me - thanks :)